client/payment-details/timeline/index.js

```diff
@@ -15,11 +15,8 @@ import mapTimelineEvents from './map-events';
 import Loadable, { LoadableBlock } from 'components/loadable';
 
 const PaymentDetailsTimeline = ( { chargeId } ) => {
-	if ( ! wcpaySettings.featureFlags.paymentTimeline ) {
```

Comment:
Moved to `client/payment-details/index.js` because otherwise the `useTimeline` hook was called conditionally